src/test/scala/org/coroutines/ast-canonicalization-tests.scala

```diff
@@ -90,7 +90,6 @@ class ASTCanonicalizationTest extends FunSuite with Matchers {
 
     val c = call(unit())
     assert(!c.resume)
-    assert(c.result == (()))
```

Comment:
It's important to figure out at least whether this will return a value, or throw an exception, so we should leave the call.